packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -143,6 +143,11 @@ const cli = {
         describe: 'Whether or not to automatically update all reference screenshots with the latest screenshots.',
         default: false,
       },
+      useSeleniumStandalonService: {
```

Comment:
I'm open to suggestions if there is a better name to use.

Reply:
seems good enough to me
```suggestion
      useSeleniumStandaloneService: {
```